package.json

```diff
@@ -64,6 +64,7 @@
     "jquery": "^3.6.0",
     "jquery-bridget": "^3.0.0",
     "jquery-ui-dist": "^1.12.1",
+    "js-base64": "^2.6.4",
```

Comment:
This is because of src/main/resources/site/parts/infoGraphics/infoGraphics.es6#L20 
`js-base64` was previously provided by `node-sass` in `ssb-component-library`